benchflow-experiment-manager/configuration.yml

```diff
@@ -85,4 +85,7 @@ testManager:
 
 experimentTaskExecutor:
   minThreads: 5
-  maxThreads: 15
\ No newline at end of file
+  maxThreads: 15
+
+testMode:
+  mockFaban: ${env.MOCK_FABAN!false?c}
```

Comment:
It is to parse the value as a boolean. I updated the issue https://github.com/benchflow/benchflow/issues/385